lms/static/scripts/postmessage_json_rpc/server.ts

```diff
@@ -171,11 +170,8 @@ export class Server {
 
   /**
    * Return a JSON-RPC response object for the given JSON-RPC request object.
-   *
-   * @param {JsonRpcRequest} request
-   * @return {Promise<JsonRpcResponse>}
    */
-  async _jsonRPCResponse(request) {
+  async _jsonRPCResponse(request: JSONRPCRequest): Promise<JSONRPCResponse> {
```

Comment:
Should this method be `private`?

```suggestion
  private async _jsonRPCResponse(request: JSONRPCRequest): Promise<JSONRPCResponse> {
```